CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 ### Added
+- Generating `alm-examples` in a pretty format ([#1793](https://github.com/operator-framework/operator-sdk/pull/1793))
```

Comment:
Let's be more descriptive here. Maybe something like:
```suggestion
- The `operator-sdk olm-catalog gen-csv` command now produces indented, newline-delimited JSON for the `alm-examples` annotation. ([#1793](https://github.com/operator-framework/operator-sdk/pull/1793))
```

Reply:
Done 👍